### PR TITLE
Start some more deployment docs.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: doc/deployment.rst [hunk 1]
```diff
@@ -92,9 +92,62 @@ terminal.  It should look something like this.
...
+
+To confirm that something's not immediately broken, you can go through the
+tests of doing ``fedmsg-logger`` and ``fedmsg-tail`` on ``hostA`` again (all
+"local").
+
+Copy that relay.py file over to ``hostB`` with ``scp /etc/fedmsg.d/relay.py
+hostB:/etc/fedmsg.d/relay.py``
+
+You should now be able to run ``fedmsg-tail`` on ``hostA`` and have it receive
+a message from ``fedmsg-logger`` on ``hostB`` and vice versa have a
```
Comment (pypingou) [inline doc/deployment.rst]: Shouldn't hostA know about hostB if you send something from hostB?

Comment (ralphbean) [inline doc/deployment.rst]: Not in this case.  ``hostB`` knows to just send messages to the ``fedmsg-relay`` running on ``hostA``.

Comment (pypingou) [inline doc/deployment.rst]: ok cool :)

**Comment by pypingou** [comment]
:+1: for me

**Comment by ralphbean** [comment]
:mega: :sunglasses:

**Pull request merged**